internal/config/config.go

```diff
@@ -0,0 +1,91 @@
+package config
+
+import (
+	"bufio"
+	"context"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/sethvargo/go-envconfig"
```

Comment:
Right now we have a mix of environment variables and CLI flags. I tend to use environment variables because I only use the server version, but we need to figure out what's the best solution for both worlds. Ideally the CLI would read configuration from some file as well. I will take a look at Viper, it seems to provide everything on one package :)